cve_bin_tool/checkers/__init__.py

```diff
@@ -2,6 +2,7 @@
 import collections
 import re
 
+from ..error_handler import InvalidCheckerError
```

Comment:
We're trying to move away from using relative imports, so this should be `cve_bin_tool.error_handler`  (and it would be great if you could fix the one below as well, which should be `cve_bin_tool.util` instead of `..util`)